specification/components/schemas/responses/GatewayTimeout.yaml

```diff
@@ -0,0 +1,19 @@
+description: |
+  Where status code 504 (Gateway Timeout) is returned then an OperationOutcome will be included in the body. An OperationOutcome will be included in the body, as detailed below.
+  
+  | Error code                         | Description                                                                                                                  |
+  | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+  | timeout                          | The API operation took longer than 9000ms. |
```

Comment:
Should the Error code be referring to the issue.details.coding.code rather than the issue.code?